packages/react/src/primitives/TextField/TextField.tsx

```diff
@@ -45,7 +45,8 @@ export const TextField: React.FC<TextFieldProps> = ({
     <Flex
       alignContent={alignContent}
       alignItems={alignItems}
-      className={classNames(ComponentClassNames.TextField, className)}
+      className={classNames(ComponentClassNames.Field, className)}
+      data-size={size}
```

Comment:
Here is the small bug. We need to apply sizing prop here, otherwise the styling defined for it in `field.scss` will not work

Reply:
Thanks for fixing this!